specification/compute/resource-manager/Microsoft.Compute/stable/2021-03-01/examples/CreateARestorePoint.json

```diff
@@ -7,81 +7,86 @@
     "vmName": "myVM",
     "api-version": "2021-03-01",
     "parameters": {
```

Comment:
Can you check or confirm if the request properties are contained in the parameters structure.

Reply:
the request properties need to be contained in the parameters structure. if i remove the params structure, the linter fails like this:
The schema 'RestorePointProperties' is considered an object but without a 'type:object', please add the missing 'type:object'.
Location: Microsoft.Compute/stable/2021-03-01/compute.json#L7676